conf/hiera.yaml

```diff
@@ -0,0 +1,10 @@
+---
+# Hiera 5 Global configuration file
+
+version: 5
+
+# defaults:
+#   data_hash: yaml_data
+hierarchy: []
```

Comment:
I suppose that'd work, but isn't it clearer to leave the commented default "as is" and then add an explicit empty entry (to show that there's a diff)? e.g.:
```
# defaults:
#    data_hash: yaml_data
# hierarchy:
#    - name: Common
#      data_hash: yaml_data
hierarchy: []
```
(nit picking for sure, but I imagine that this file will be very visible to end users)